checkpointctl.go

```diff
@@ -122,7 +130,9 @@ func show(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	if err := untarFiles(input, dir, metadata.SpecDumpFile, metadata.ConfigDumpFile); err != nil {
+	// List of files required from the checkpoint archive
+	files := []string{metadata.SpecDumpFile, metadata.ConfigDumpFile, metadata.CheckpointDirectory}
```

Comment:
This works, but this is against the goal I had in mind with the change to only unpack the needed files.

If the user requests the process tree, only then should the corresponding files be unpacked. According to the test case below only `pstree.img` and `core-*.img` files are needed.

Reply:
Got it, I'll change this. But do note that we will have to add more files as we go along, for the other explore types.

Reply:
I have changed it to include the individual files that each feature requires. For now, we are only unarchiving `pstree.img` and `core-*.img` for the process tree. A nifty feature of `IncludeFiles` with `tar` is that it allows us to specify wildcards!